R/log.R

```diff
@@ -285,8 +288,39 @@ log_write <- function(file = NA,
                         write_log_element("log_path", "Log path: "))
 
    writeLines(cleaned_log_vec,
-              con = file.path(get_log_element("log_path"),
-                              get_log_element("log_name")))
+                 con = file.path(get_log_element("log_path"),
+                                 get_log_element("log_name")))
+   if (include_rds){
+      rds_fields <- c(
+         "end_time", "start_time", "run_time", "user", "hash_sum",
+         "log_path", "log_name", "file_path", "file_name",
+         "unapproved_packages_functions", "errors", "warnings"
+      )
+      log_options <- as.list(getOption('log.rx'))
+      cleaned_log_list <- purrr::map2(
```

Comment:
I haven't 100% convinced myself, but I'm wondering if we would just return everything to the rds rather than NULL out sections.  Those writing the log and those consuming these rds files might be different people with different use cases.